src/drivers/px4fmu/fmu.cpp

```diff
@@ -1026,74 +1026,75 @@ PX4FMU::cycle()
 			}
 		}
 
-		/* can we mix? */
-		if (_mixers != nullptr) {
+	}
 
-			size_t num_outputs;
+	_cycle_timestamp = hrt_absolute_time();
```

Comment:
The solution to that is to either also watch the armed topic or enforce a minimum rate. Obviously the output is the combination of actuator and armed and so both need to be watched.